src/components/MultipleValueTextInput.tsx

```diff
@@ -64,6 +66,24 @@ function MultipleValueTextInput({
 		setValue('');
 		onItemAdded(value, newValues);
 	};
+	const handleItemsAdd = (addedValues: string[]) => {
+		let uniqueValues: string[] = []
```

Comment:
A Javascript tidbit for the future - since `addedValues` will simply be an array of strings, we can use the following to remove duplicates from it:

```
let uniqueValues = Array.from(new Set(addedValues));
```

which would reduce lines 70-75 down to one line. 😄 

We also could (and maybe should?) do the same thing on line 77 to make sure there's no duplicates between the pasted values and the existing array of values.

```
const newValues = Array.from(new Set([...values, ...uniqueValues]));
```

(That last bit is the "spread" operator syntax; happy to provide more resources on that too.)

Reply:
oh let me tc of this real quick

Reply:
jeez, I was so caught up in trying to get the biz logic to work within react/js, that I forgot i was basically implementing a set in lines 70-75. so dumb!!! Thanks hehe

Reply:
you actually caught a bug there in line 77 too where we would have had duplicates if values and uniqueValues had the same item in them. good eye

Reply:
I wouldn't call it dumb. :D Your approach was valid (and the syntax I described is comparatively new in JS). But I do like the conciseness of the more modern JS array functions (as long as they don't interfere with readability).

Reply:
ya kinda like what java does with collections/streams. i dig it

Reply:
ok I took your advice and had to slightly tweak it 
I replaced lines 70-75 with
 `let uniqueValues = Array.from(new Set(addedValues.filter(elm => elm && !values.includes(elm))));`
 
 because in addition to making sure the elements are unique, I also need to remove empty/null and also remove items that are already in the values array. I need to do this here because I then use uniqueValues to call onItemAdded and so I cannot have existing values in this array. Let me know if there is a better way to do this

Reply:
Nope, that looks great!